File: src/main/java/no/nav/syfo/controller/domain/adresse/StrukturertAdresse.java
package no.nav.syfo.controller.domain.adresse;


import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;
import java.io.Serializable;

@Data
@Accessors(fluent = true)
@EqualsAndHashCode
public class StrukturertAdresse implements Serializable {
    public String landkode;
    public String tilleggsadresse;
    public Gateadresse gateadresse;
    public Matrikkeladresse matrikkeladresse;
    public PostboksadresseNorsk postboksadresseNorsk;
}

File: src/main/kotlin/no/nav/syfo/controller/user/domain/adresse/StrukturertAdresse.kt
package no.nav.syfo.controller.user.domain.adresse

import java.io.Serializable

open class StrukturertAdresse(
    val landkode: String? = null,
    val tilleggsadresse: String? = null,
    var gateadresse: Gateadresse? = null,
    val matrikkeladresse: Matrikkeladresse? = null,
    var postboksadresseNorsk: PostboksadresseNorsk? = null
) : Serializable
